neuro_flow/batch_runner.py

```diff
@@ -370,6 +374,7 @@ async def _process_topo(
         attempt: Attempt,
         prefix: FullID,
         topo: graphlib.TopologicalSorter[FullID],
+        ready: List[FullID],
```

Comment:
After the executor class extraction `ready` will be an executor private property (among with topos, started, finished, skipped, etc.) but for now better to pass it directly as a function argument